bin/bundler.rs

```diff
@@ -79,15 +79,30 @@ fn main() -> Result<()> {
 
                 let chain_id = eth_provider.get_chainid().await?;
 
-                let wallet = Wallet::from_file(opt.mnemonic_file, chain_id)?;
-                info!("{:?}", wallet.signer);
-
                 let eth_provider =
                     Arc::new(Provider::<Http>::try_from(opt.eth_client_address.clone())?);
 
-                let _bundler = Bundler::new(wallet);
+                let uopool_grpc_client = UoPoolClient::connect(format!(
+                    "http://{}",
+                    opt.uopool_opts.uopool_grpc_listen_address
+                ))
+                .await?;
+                for entry_point in opt.entry_points.iter() {
+                    let wallet = Wallet::from_file(opt.mnemonic_file.clone(), chain_id)?;
+                    info!("{:?}", wallet.signer);
+                    let _bundler = Bundler::new(
+                        wallet,
+                        opt.bundler_opts.beneficiary,
+                        uopool_grpc_client.clone(),
+                        opt.bundler_opts.bundle_interval,
+                        *entry_point,
+                        opt.eth_client_address.clone(),
+                        opt.bundler_opts.chain_id,
```

Comment:
Is this necessary? `chain_id` is already defined on line 80, no need to pass it as additional parameter for bundler.

Reply:
Oh. Sorry. I missed that.